### DISCUSSION TITLE
DIY - OAuth broker application - still an option ?

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/domino-online-meeting-integration#159] [Q&A]
Hello,
HCL told me that hosting the OAuth broker application is not supported anymore.
But still customers are craving to host this app in their own DC to avoid HCL/3rd party progressing their meeting tokens.
Does anybody run this broker app in their DC ? If yes: Is there a change to get an instructions on how to handle this ?
Or is this really a dead topic ?

Regards,
Jan
*I was wondering what happens when HCL servers have an outage. Will DOMI in V12 Notes clients just stop to function ?
Just for a backup-solution hosting the broker app on one of our own server would be a good thing.

**Comment by paulswithers** [comment]
Sorry for the delay in update. We have failover in the HCL servers for retrieving tokens, so it shouldn't be a problem you have to worry about. If customers want to host the web app in a data center, that is covered in the documentation, https://opensource.hcltechsw.com/domino-online-meeting-integration/modules_webapp/overview/. I received questions about how to set it up, so I believe some business partners are using it. But I don't have specific customers to point you towards.
